baton/handler.go

```diff
@@ -14,6 +15,8 @@ func Handler() *mux.Router {
 	return m
 }
 
+var i = 0
```

Comment:
If both instances are using the same port, yes. Otherwise, the client won't use both and it will be fine.

To be clear, this is an identifier per session not per user.